pythermalcomfort/models/phs.py

```diff
@@ -133,6 +136,12 @@ def phs(
         )
         print(result.t_re)  # [37.5 43.4]
     """
+
+    if model not in [Models.iso_7933_2004.value, Models.iso_7933_2023.value]:
+        raise ValueError(
+            "Model should be '7933-2004' or '7933-2023'. Please check the documentation."
```

Comment:
change to f"Model should be '{Models.iso_7933_2004.value}' or '{Models.iso_7933_2023.value}'. Please check the documentation."